web/packages/teleport/src/Users/UserReset/UserReset.story.tsx

```diff
@@ -16,40 +16,69 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import { Meta } from '@storybook/react';
+
+import { Attempt } from 'shared/hooks/useAttemptNext';
+import { useEffect, useState } from 'react';
+
+import cfg from 'teleport/config';
+
 import { UserReset } from './UserReset';
 
-export default {
-  title: 'Teleport/Users/UserReset',
+type StoryProps = {
+  status: 'processing' | 'success' | 'error';
+  isMfaEnabled: boolean;
+  allowPasswordless: boolean;
 };
 
-export const Processing = () => {
-  return <UserReset {...props} attempt={{ status: 'processing' }} />;
+const meta: Meta<StoryProps> = {
+  title: 'Teleport/Users/UserReset',
+  component: Story,
+  argTypes: {
+    status: {
+      control: { type: 'radio' },
+      options: ['processing', 'success', 'error'],
+    },
+  },
+  args: {
+    status: 'processing',
+    isMfaEnabled: true,
+    allowPasswordless: true,
+  },
 };
 
-export const Success = () => {
-  return <UserReset {...props} attempt={{ status: 'success' }} />;
-};
+export default meta;
+
+export function Story(props: StoryProps) {
+  const statusToAttempt: Record<StoryProps['status'], Attempt> = {
+    processing: { status: 'processing' },
+    success: { status: 'success' },
+    error: { status: 'failed', statusText: 'some server error' },
+  };
+  const [, setState] = useState({});
+
+  useEffect(() => {
+    const defaultAuth = structuredClone(cfg.auth);
+    cfg.auth.second_factor = props.isMfaEnabled ? 'on' : 'off';
+    cfg.auth.allowPasswordless = props.allowPasswordless;
+    setState({}); // Force re-render of the component with new cfg.
+
+    return () => {
+      cfg.auth = defaultAuth;
+    };
+  }, [props.isMfaEnabled, props.allowPasswordless]);
```

Comment:
I was wondering how we usually "mock" cfg in the Web UI and this is what I found in `e/web/teleport/src/AccessMonitoring/Timeframe.story.tsx`. https://github.com/gravitational/teleport.e/blob/ab7c9fca11b43e3a3023fe4fd412b2d4e58377e6/web/teleport/src/AccessMonitoring/Timeframe.story.tsx#L17-L25